db-es-audit/test/find_granule_counts_test.py

```
@@ -0,0 +1,300 @@
+            self.assertEqual(
+                find_granule_counts.missing_efs_file_name("P1"),
+                "/mnt/efs/P1/P1_granule_counts_mismatch.json",
+            )
+        finally:
+            find_granule_counts.EFS_PATH = old
+
+
+class FindGranuleCountsMismatchTests(unittest.TestCase):
+    def test_find_granule_counts_mismatch_creates_file_runs_compare_and_uploads(self):
+        old_efs = find_granule_counts.EFS_PATH
+        find_granule_counts.EFS_PATH = "/tmp/efs/"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/efs/"

(S108)